add and update sites

alyssamichelle opened the pull request.

lalithpolepeddi commented:
Thanks @alyssamichelle! Two things:
1. For jsonschema.net, in its project slug please replace the `.` with a `-` so we can have project URLs without extra `.`s. Three places to make this change:
    * Update the post name to **_posts/2021-1-11-jsonschema-net.md**
    * In the post, update the slug name to `slug: jsonschema-net`
    * Update the image directory to **assets/images/projects/jsonschema-net**
1. Is there a post that goes with the image in **assets/images/projects/adore-me/thumb.jpg**?

Then give it a test locally (`$ bundle exec jekyll serve`) to make sure everything works and let me know!

alyssamichelle commented:
Thanks @lalithpolepeddi , I was wondering how to treat the `.`. I fixed the issues with jsonschema.net and added the missing post for adoreme. Looks like it is all rendering:

![image](https://user-images.githubusercontent.com/1058831/105773774-11fd2500-5f2a-11eb-933d-29c655a26a9b.png)
![image](https://user-images.githubusercontent.com/1058831/105774320-c303bf80-5f2a-11eb-98d9-1c5c55fd3796.png)

lalithpolepeddi commented:
Looks great, thanks for the updates!
